cypress/integration/smoke.spec.ts

```diff
@@ -69,14 +68,14 @@ const addTablePanel = (q: string) => {
     },
   });
 
-  e2e.flows.explore({
-    matchScreenshot: false,
-    timeRange: {
-      from: '2001-01-31 19:00:00',
-      to: '2016-01-31 19:00:00',
-    },
-    queriesForm: () => fillQuery(query),
-  });
+  // e2e.flows.explore({
```

Comment:
I had to remove the explore flow as it seems it's not supported by e2e anymore? 🤔

Reply:
Seems reasonable to me to remove rather than comment out. If we decide we want to figure out how to add this back in we can find it in the git history, what do you think?

Reply:
Yes, I was wondering about that, keeping broken code commented out does not help 😉 I'll remove it!